.github/workflows/diff.yml

```diff
@@ -1,19 +1,18 @@
 name: 'Create ROBOT diffs on Pull requests'
 
-on:
-  # Triggers the workflow on pull request events for the master branch
-  pull_request:
-    branches: [ master ]
-    paths:
-      - 'src/ontology/uberon-edit.obo'
+on: 
 
-  # Allows you to run this workflow manually from the Actions tab
-  workflow_dispatch:
+  issue_comment:
+        types: [created]
```

Comment:
Excellent, so this fires whenever a comment is made (what does `created` means, as opposed to.. `editied`?), and `if: ${{ github.event.issue.pull_request }}` makes sure that the action is only run whenever the issue is on a pull request?

Reply:
yes, it will fire when any comment is created (as opposed to edited or deleted), and will skip everything if the comment isn't in a PR.

Reply:
cool!